README.md

```
@@ -186,3 +186,10 @@ NuGet package versions are calculated from a combination of tags and commit mess
 * `+semver: minor` or `+semver: feature` - update minor version number, reset patch to 0 (so 2.3.1 would become 2.4.0)
 * Anything else, including no `+semver` lines at all - update patch version number (so 2.3.1 would become 2.3.2)
     * If you want to include `+semver` lines, then `+semver: patch` or `+semver: fix` are the standard ways to increment a patch version bump, but the patch version will be bumped regardless as long as there is at least one commit since the most recent tag.
+
+### Run benchmarks
+
+docs: https://benchmarkdotnet.org/articles/guides/console-args.html
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)